projects/mysadna/.firebase/hosting.ZGlzdA.cache

```diff
@@ -0,0 +1,38 @@
+mysadna2/index.html,1550673544454,d112ea33158ff5e6920baa100808b97c8607891fa9d506693f5abc68c2cba6ea
```

Comment:
:smile: Yeah.. It happens sometimes.
What you need to do:
1) Remove `/.firebase` and `/public` folders
2) Commit changes
3) Add the folders to gitignore
4) Commit changes again
5) Optional: Make rebase last 2 commits